etc/systemd/system/proxysql.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=High Performance Advanced Proxy for MySQL
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/proxysql -c /etc/proxysql.cnf
+PIDFile=/var/run/proxysql/proxysql.pid
+#ExecReload=
+
+ExecStop=/bin/kill ${MAINPID}
```

Comment:
This is clearly optional. _systemd_ will send kill according to `KillMode`.
If you don't want to send kill to the entire _cgroup_, you can use `KillMode=process`.